app/controllers/members_controller.rb

```diff
@@ -132,7 +143,7 @@ def person_params
               credentials: [:token, :expires_at, :expires ],
               info:        [:email, :name, :image ] ]
     ).tap{ |person_attrs| handle_empty_contact_info(person_attrs) }
-      .tap { |person_attrs| maybe_set_role(person_attrs) }
+      .tap { |person_attrs| maybe_set_memberships(person_attrs) }
```

Comment:
sure. like `tap { |person_attrs| format_person_attrs }` which dispatches to `maybe_set_role` and `maybe_set_memberships`?

only thing i think is important is to keep the two helpers distinct since they do two different things.

Reply:
whoops. forgot to field this one. think it's okay to leave on the table for now. if we wind up with a bunch of `tap`s piling up (ie: one more than this), let's group them into one.